docs-js/langchain/orchestration.mdx

```diff
@@ -71,6 +71,41 @@ const response = await client.invoke(history, {
 });
 ```
 
+## Streaming
+
+The client supports streaming responses for chat completion requests.
+Use the `stream()` method to receive a stream of chunk responses from the model.
+
+By default, the last chunk contains the `finish_reason` and token `usage` information.
+The orchestration service doesn't support multiple choices during streaming.
+
+```ts
+const orchestrationConfig: LangchainOrchestrationModuleConfig = {
+  llm: {
+    model_name: 'gpt-4o'
+  }
+};
+
+const client = new OrchestrationClient(orchestrationConfig);
+const response = await client.stream([
+  {
+    role: 'user',
+    content:
+      'Write a 100 word explanation about SAP Cloud SDK and its capabilities'
+  }
+]);
+
+let finalResult: AIMessageChunk | undefined;
+
+for await (const chunk of response) {
+  console.log(chunk.content);
+  finalResult = finalResult ? finalResult.concat(chunk) : chunk;
+}
+
+console.log(finalResult?.response_metadata?.finishReason);
+console.log(finalResult?.usage_metadata);
+```
+
 ## Resilience
```

Comment:
[req] In the `Timeout` section, could you add `stream` method behind the existing `invoke`.

[q] Currently, timeout won't be retried in `invoke` but in `stream` it will... I feel it is an implementation issue, since timeout should be retried in `invoke` as well... See https://github.com/langchain-ai/langchainjs/issues/2706. No strong opinion.